data-structures-stacks/is-empty.js

```diff
@@ -0,0 +1,5 @@
+/* exported isEmpty */
+
+function isEmpty(stack) {
+  return stack.peek() === undefined ? true : false;
```

Comment:
Here you can just return the condition since it will evaluate to `true` or `false`

```JAVASCRIPT
return stack.pop() === undefined;
```